src/pages/Messages.tsx

```diff
@@ -1,5 +1,68 @@
+import { Box, Button, Divider, List, Stack, Typography } from "@mui/material";
+import ChatIcon from "@mui/icons-material/Chat";
+import IconButton from "@mui/material/IconButton";
+import SearchBar from "../components/Messages/SearchBar";
+import MessagesList from "../components/Messages/MessagesList";
+
+const styles = {
+  button: {
+    borderRadius: 10,
+    marginTop: 1,
+  },
+  messagesHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    paddingTop: 2,
+    paddingRight: 2,
+    paddingLeft: 2,
+    paddingBottom: 0,
+  },
+  messageListContainer: {
+    width: "30%",
+  },
+  root: {
+    width: "100%",
+  },
+  selectMessageContainer: {
+    alignItems: "flex-start",
+    display: "flex",
+    flexDirection: "column",
+    margin: "auto",
+    width: "50%",
+  },
+};
+
 const Messages = () => {
-  return <h1 style={{ margin: "auto" }}>Messages</h1>;
+  return (
+    <Stack
+      direction="row"
+      sx={styles.root}
+      divider={<Divider orientation="vertical" />}
+    >
+      <Box sx={styles.messageListContainer}>
+        <Box sx={styles.messagesHeader}>
+          <Typography variant="h6">Messages</Typography>
+          <IconButton>
+            <ChatIcon />
+          </IconButton>
+        </Box>
+        <SearchBar />
+        <List component="div">
+          <Divider />
+          <MessagesList />
+        </List>
+      </Box>
+      <Box sx={styles.selectMessageContainer}>
+        <Typography variant="h6">Select a Message</Typography>
+        <Typography>
+          Choose one of your existing conversations or start a new one!
+        </Typography>
+        <Button variant="contained" sx={styles.button}>
+          New Message
+        </Button>
+      </Box>
```

Comment:
I think we should add a divider on the other side of the Messages Box, but that would mean we need to make changes elsewhere (in the `styles.selectMessageContainer`). 

I tried it out, and the changes we would need to make it fixing the margin left and right to a certain amount of pixels (`marginLeft: "40px"` , `marginRight: "40px"`) and also decreasing the `width` to `40%`.

If you agree with the design change, but wanna do it in a different way, feel free to!

```suggestion
      </Box>
      <Divider  orientation="vertical" />
```